test/ci-plugin-e2e/insights-ci.sh

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC1003
-set -xeo pipefail
+set -e
```

Comment:
I think `-o pipefail` is important, but I've never fully understood what it does